libraries/libmagic/make.sh

```diff
@@ -0,0 +1,76 @@
+PKGURL=https://github.com/threatstack/libmagic/archive/5.18.tar.gz
+PKGHASH=ce734a1cc24bddbfd8b21f4ffdfe721ab74eeed9
+
+package_download $PKGURL $PKGHASH
+package_patch
+
+EXTRACONF=
+
+if [ "$SYS_PLATFORM" != "$SYS_HOSTPLATFORM" ]; then
+    EXTRACONF="$EXTRACONF --host=$SYS_ARCH"
+fi
+
+# pretend libgnurx exists by masking libpcre
+
+test -L $SYS_PREFIX/include/regex.h || ln -s pcreposix.h $SYS_PREFIX/include/regex.h
+
+# Mabye instead of fooling the package, it should be patched
```

Comment:
Not quite sure about this either. Why not include that library if you depend on it?

Reply:
Two reasons:

1. At least I need `pcre` anyway and I see limited reasons to have two libraries implementing the same functionality.

2. I'd guess there might be license compatibility issues when linking against a GPLed library and `libgnurx` is AFAIK.